### ISSUE TITLE
Chunk size (in older 1.1.x release)

### ISSUE THREAD

**Issue by pczerkas** [facile-it/paraunit#163]
I would like to add some new feature in 1.1.1 codebase, will this be possible to commit it into your project? I'm using paraunit to run stock Magento 2 integration tests suite (PHPUnit 9.1.5) and there is some overhead with bootstrapping each test class.

New option `--chunk-size` (default to 1) when set to larger than 1 would change paraunit runner behaviour to create filtered "phpunit_[0|1|...].xml" file per process with < testsuite > multiple < file > tags chunked from master phpunit.xml file that are further stuffed in parallel to PHPUnit. This way bootstrapping for tests would be reduced.

**Comment by Jean85** [comment]
I've briefly reviewed the PR and it seems a good addition, thanks! I'll try to fix the CI for the old branch, so we can safely merge it later; I'll also work to port up in the latest version it too.

**Comment by Jean85** [comment]
Oh, I just noticed that you wanted this released in the 1.1.x series, but the PR is targeted to 1.x, the latest branch. We would need to do the opposite then, to backport the feature; it's doable, please just don't delete your fork after the merge, we could open another PR with another target later.

**Comment by pczerkas** [comment]
> Oh, I just noticed that you wanted this released in the 1.1.x series, but the PR is targeted to 1.x, the latest branch. We would need to do the opposite then, to backport the feature; it's doable, please just don't delete your fork after the merge, we could open another PR with another target later.

Yes, I was trying to create PR for 1.1.1 tag as base, but turned out github does not allow to make PRs to past tag. Maybe new branch (for 1.1.x series) should be created in paraunit repo for such operation?

**Comment by Jean85** [comment]
Yes, I'll do it as soon as the PR is ready. As of now, I'm having some issues with the build in #165 though.
